product_category_product_qty/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module extends the functionality of product module to display
+the quantity of products and variants for the product categories.
```

Comment:
As far I understand here only variant are computed, also suggest wording  to avoid to confused user with stock quantities.

```suggestion
the number of products for the product categories.
```